### DISCUSSION TITLE
MSS310 Realtime Power stats?

### DISCUSSION THREAD

**Discussion by solarjones** [krahabb/meross_lan#295] [General]
Hi,
I'm using a pair of MSS310 (h/w: 6.0.0.0 with f/w: 6.3.21) smart monitor plugs and the integration works (I'm using http option) for controlling the on/off functionality. 

However I don't seem to be able to get realtime Power data.  The Moross app shows the correct wattage in real-time when I switch devices on/off, just not in HA. There appears to be a big delay (~20 minutes) before HA reports Power values. 

Am I missing something? What's the refresh frequency of this data?

Thanks!

**Comment by krahabb** [comment]
Hello @solarjones,
on HTTP it should refresh evry 30 secs but I'm investigating a possible glitch in the protocol management logic.
Could you try setting the device configuration entry protocol to 'HTTP' (explicitly) and see if it fixes this?

**Reply by solarjones** [reply]
Hi @krahabb, 
Thanks for th suggestion. I'll try that and see how it responds.
